### PR TITLE
Register in Send To menu instead of plugins toolbar

### PR DESCRIPTION
**Pull request by mosseb**

### CODE REVIEW

File: main.qml [hunk 1]
```diff
@@ -17,8 +25,8 @@ PainterPlugin {
 		 	}
 		 	alg.settings.setValue("padding", false);
 		}
-		// create the button
-		alg.ui.addWidgetToPluginToolBar("exporter.qml");
+		var sendtoAction = alg.ui.addAction(alg.ui.AppMenu.SendTo, "Send to Ps", qsTr("Export to Photoshop"), Qt.resolvedUrl("icons/Photoshop_idle.svg"), Qt.resolvedUrl("icons/Photoshop_hover.svg"));
```
Comment (PhilippeHermite) [inline main.qml:28]: qsTr("Send to Ps") ???

File: main.qml [hunk 2]
```diff
@@ -29,4 +37,110 @@ PainterPlugin {
   ConfigurePanel {
     id: configurePanel
   }
+
+  QtObject {
+	property bool loading: false
```
Comment (PhilippeHermite) [inline main.qml:42]: spaces or tabs? ^^

File: main.qml [hunk 3]
```diff
@@ -29,4 +37,110 @@ PainterPlugin {
...
+    title: qsTr("Please locate Photoshop...")
+    nameFilters: [ "Photoshop files (*.exe *.app)", "All files (*)" ]
+    selectedNameFilter: "Executable files (*)"
+    onAccepted: {
+      alg.settings.setValue("photoshopPath", alg.fileIO.urlToLocalFile(fileUrl.toString()));
+      internal.launchExportDialog()
+    }
+  }  
+
+  function sendToTriggered() {
```
Comment (PhilippeHermite) [inline main.qml:137]: move into internal??

**Comment by mosseb** [comment]
created pull request #48 instead

**Pull request closed without merging**